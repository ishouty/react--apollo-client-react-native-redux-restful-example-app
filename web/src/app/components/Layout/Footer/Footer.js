import React, { Component } from 'react'
import { FooterText } from '../../../constants/text/text'

export default class Footer extends Component {

    render() {

        return (

            <div className="footer">
                <div>
                    {FooterText.rightReserved}
                </div>
            </div>

        )
    }

}
