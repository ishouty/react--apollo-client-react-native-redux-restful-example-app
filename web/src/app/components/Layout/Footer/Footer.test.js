import React, { Component } from 'react'
import Footer from './Footer'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'


describe('Footer', ()=> {

    test('Component Should render', ()=> {

        const component = shallow(<Footer/>)
        expect(component.find('.footer')).toHaveLength(1)

    })

    test('Render and match snapshot', ()=> {

        const component = shallow(<Footer/>)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()

    })

})