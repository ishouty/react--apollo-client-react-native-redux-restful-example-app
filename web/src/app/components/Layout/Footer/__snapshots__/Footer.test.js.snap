// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Render and match snapshot 1`] = `
<div
  className="footer"
>
  <div>
    Andy Nguyen @ ishouty all rights reserved
  </div>
</div>
`;
